excercise-e-shop/assets/scss/base/_variable.scss

```diff
@@ -0,0 +1,7 @@
+$base-color: #FBB03B;
```

Comment:
not only color, font-size, font-family, break-points, every reusable-value need to be defined by variables